docs/src/assets/scss/components/search.scss

```diff
@@ -112,7 +111,6 @@
 }
 
 .search .search-results__item__title {
-    font-size: var(--step-0);
```

Comment:
This is intentionally duplicate. The first is used when the browser support variables and the second is used when the browser does not support variables.

Reply:
Added intentional properties and removed only duplicate properties & fixed typos.